cesium/writer_behavior_test.go

```diff
@@ -42,14 +42,15 @@ var _ = Describe("Writer Behavior", func() {
 			})
 			AfterAll(func() {
 				Expect(db.Close()).To(Succeed())
+				_ = cleanUp
 				Expect(cleanUp()).To(Succeed())
 			})
 			Describe("Happy Path", func() {
 				Context("Indexed", func() {
 					Specify("Basic Write", func() {
 						var (
-							basic1      = GenerateChannelKey()
-							basic1Index = GenerateChannelKey()
+							basic1      = testutil.GenerateChannelKey()
```

Comment:
Any reson why you removed the `.` here. I prefer importing testutils with `.` imports

Reply:
not really